.flake8.ini

```diff
@@ -1,5 +1,6 @@
 [flake8]
-max-line-length = 120
+max-line-length = 150
+ignore = W191
```

Comment:
@vaibhavsingh97 What is W191?

Reply:
Tabs to spaces

Reply:
Flake8 looks okay. What do @S2606 think?